python/lambdas/copy-files-from-tdr/lambda_function.py

```diff
@@ -0,0 +1,84 @@
+import os
+import json
+import boto3
+
+s3_client = boto3.client('s3')
+sqs_client = boto3.client('sqs')
+
+
+def lambda_handler(event, context):
+    destination_bucket = os.environ["DESTINATION_BUCKET"]
+    destination_queue = os.environ["DESTINATION_QUEUE"]
+    for record in event['Records']:
+        body = json.loads(record['body'])
+        file_id = body['fileId']
+        source_bucket = body['bucket']
+        file_location = copy_object(destination_bucket, file_id, source_bucket)
+        copy_object(destination_bucket, f"{file_id}.metadata", source_bucket)
+
+        sqs_client.send_message(QueueUrl=destination_queue, MessageBody=json.dumps({'location': file_location}))
+
+
+def copy_object(destination_bucket, s3_key, source_bucket):
+    try:
+        response = s3_client.head_object(Bucket=source_bucket, Key=s3_key)
+        file_size = response['ContentLength']
+
+        if file_size > 5 * 1024 * 1024 * 1024:
+            return multipart_copy(source_bucket, destination_bucket, s3_key, file_size)
+        else:
+            return standard_copy(source_bucket, destination_bucket, s3_key)
+
```

Comment:
1. Since both of these functions return `f"s3://{destination_bucket}/{s3_key}"`, should we just add a `return a f"s3://{destination_bucket}/{s3_key}"` here instead of in each function?